app/src/main/java/org/mozilla/focus/history/HistoryItemAdapter.java

```diff
@@ -155,7 +155,7 @@ public int getItemCount() {
     @Override
     public void onClick(View v) {
         final int position = mRecyclerView.getChildLayoutPosition(v);
```

Comment:
Would it be better if we use "getChildAdapterPosition"?

Reply:
That would be better. I would revise "getChildLayoutPosition" to "getChildAdapterPosition" and force push it.